frame/support/src/traits.rs

```diff
@@ -216,6 +216,19 @@ impl<A> FindAuthor<A> for () {
 	}
 }
 
+/// A trait to get the author of the block.
+pub trait Author<AccountId> {
+	/// Return the account of the author of the block.
+	fn author() -> AccountId;
+}
+
+/// Return the default account id as author.
+impl<A: Default> Author<A> for () {
```

Comment:
I think we should remove this implementation for `()`. 

I think people could use `()` in their runtime and maybe fuck something up.

We could provide a type `TestAuthor` or whatever to make testing easier, but to remove the footgun.

Reply:
Yes and I fooled myself, node-template is wrong in this PR currently

Reply:
Done

Reply:
Done